CHANGELOG-1.1.md

```diff
@@ -1,3 +1,46 @@
+# TiDB Operator v1.1.0 Release Notes
+
+This is the GA release of TiDB Operator 1.1, which focuses on the usability, extensibility and security.
+
+See our official [documentation site](https://pingcap.com/docs/tidb-in-kubernetes/stable/) for new features, guides, and instructions in production, etc.
+
```

Comment:
Add a step to guide user to upgrade from 1.0 to 1.1?

Reply:
do we have a doc about this in docs-tidb-operator? we can add a link here.

Reply:
add a section "Upgrade from v1.0.x"